app/models/media/audio.rb

```diff
@@ -0,0 +1,4 @@
+class Media::Audio < Media::Object
+  validates_attachment_content_type :item, content_type: [%r{\Aaudio/.*\Z}, 'video/webm']
+  validates_attachment_file_name :item, matches: /(mp3|ogg|webm|wav)\Z/i
```

Comment:
I'm not sure if there's any particular reason, it's omitted in the documentation examples for paperclip's file name validations, but it could be an oversight rather than intentional on their part.